README.rst

```diff
@@ -135,3 +179,12 @@ or use ``uname -m`` inside the container.)
    :target: https://github.com/fangohr/octopus-in-docker/actions/workflows/debian-develop.yml
 
 
+Status
```

Comment:
one suggestion might be to move this to the top, that way you can see the pipeline status without scrolling. Either way is fine.

Reply:
We had it at the top in the beginning, but I thought it is of little use to the users of the container.